content/webapp/pages/stories.js

```diff
@@ -179,15 +184,14 @@ export class StoriesPage extends Component<Props> {
               'row bg-cream row--has-wobbly-background': true,
             })}
           >
-            <Space
-              v={{ size: 'l', properties: ['margin-bottom'] }}
-              className="container"
-            >
-              <div className="grid">
-                <div className={grid({ s: 12, m: 12, l: 12, xl: 12 })}>
-                  <StoryPromoFeatured item={articles[0]} />
-                </div>
-              </div>
+            <Space v={{ size: 'xl', properties: ['margin-bottom'] }}>
+              <FeaturedCard
```

Comment:
Could it feel a bit nicer to have:
```JS
<FeaturedCardArticle article={article} />

// FeatureCardArticle.js
const FeatureCardArticle = ({ article }) => {
  const featureCardProps = convertContentToFeatureCardProps
  return <FeaturedCard props={featureCardProps}><FeatureCardArticleBody article={article} /></FeaturedCard>
}
</FeaturedCardArticle>
```

I only ask as the shape we've given here will always be the shape, so why not define it fully?

Reply:
Hmmmm… yeah, I think that makes sense.

Reply:
My question I suppose is, are we ever going to have any other way to write the Article card, and if not, shouldn't we just make that the case.

Reply:
We could maybe make an e.g. `CardArticleBody` with a prop to determine whether or not it's "featured", then work this back into the existing `StoryPromo`, so that we no longer have three separate components for the regular exhibition/event/story cards, but a similar situation to this `FeaturedCard` stuff.

I might leave that as a `TODO` possibility, though.